src/components/v5/shared/SearchSelect/SearchSelect.tsx

```diff
@@ -68,16 +69,22 @@ const SearchSelect = React.forwardRef<HTMLDivElement, SearchSelectProps>(
     const content = (
       <>
         <div className="mb-6">
-          <SearchInput onInput={onInput} />
+          {isMobile && hideSearch ? (
```

Comment:
hideSearch prop is a bit bad, because it doesn't say that it should be hidden on mobile